index.d.ts

```diff
@@ -532,6 +536,13 @@ declare module 'plaid' {
     getCategories(cb: Callback<CategoriesResponse>): void;
 
     resetLogin: AccessTokenFn<ResetLoginResponse>;
+
+    // sandboxCreatePublicToken(String, Array<sring>,Object?, Function)
+    sandboxPublicTokenCreate(institutionId: string,
+                    initialProducts: Array<string>,
+                    options: Object,
+                    cb: Callback<SandboxPublicTokenCreateResponse>
+    ): Promise<SandboxPublicTokenCreateResponse>;
```

Comment:
we need to define two flavors of this function, one for the CPS style, and one for the promisified variant

```
// sandboxCreatePublicToken(String, Array<sring>,Object?, Function)
sandboxPublicTokenCreate(
  institutionId: string,
  initialProducts: Array<string>,
  options: Object,
  cb: Callback<SandboxPublicTokenCreateResponse>,
): void;

// sandboxCreatePublicToken(String, Array<sring>,Object?, Function)
sandboxPublicTokenCreate(
  institutionId: string,
  initialProducts: Array<string>,
  options: Object,
): Promise<SandboxPublicTokenCreateResponse>;
```

when they are both defined, typescript will pick the one that compiles.

Reply:
Thanks @ntindall - should be ⚡️'d!